test/ui/starter-select.test.ts

```diff
@@ -468,7 +468,7 @@ describe("UI - Starter select", () => {
     let optionSelectUiHandler: OptionSelectUiHandler | undefined;
     await new Promise<void>((resolve) => {
       game.onNextPrompt("SelectStarterPhase", UiMode.OPTION_SELECT, () => {
-        optionSelectUiHandler = game.scene.ui.getHandler() as OptionSelectUiHandler;
+        optionSelectUiHandler = game.scene.ui.getCurrentHandler() as OptionSelectUiHandler;
```

Comment:
NIT: Use generic param `<T>`

```suggestion
        optionSelectUiHandler = game.scene.ui.getCurrentHandler<OptionSelectUiHandler>();
```